src/gui/entry/EntryView.cpp

```diff
@@ -121,6 +121,20 @@ void EntryView::keyPressEvent(QKeyEvent* event)
 #endif
     }
 
+    int last = m_model->rowCount()-1;
```

Comment:
Spaces please. ;-)

Same for if below.

Reply:
Fixed with a rebase